src/sentry/utils/cloudfunctions.py

```diff
@@ -36,6 +37,9 @@ def function_pubsub_name(funcId):
 
 
 def create_function_pubsub_topic(funcId):
+    logging.info(f"Created topic {function_pubsub_name(funcId)}")
+    logging.info(f"Project name is {SENTRY_FUNCTIONS_PROJECT_NAME}")
+    logging.info(f"Region name is {SENTRY_FUNCTIONS_REGION}")
     publisher = PublisherClient()
     publisher.create_topic(name=function_pubsub_name(funcId))
```

Comment:
@vpandiarajan20 I'd recommend just logging the full output of `function_pubsub_name`

Reply:
well I guess you might want to validate the region as well but it might make sense to just add another log message where it's actually used